includes/Admin/Site_Health.php

```diff
@@ -0,0 +1,244 @@
+	 */
+	private $experiments;
+
+	/**
+	 * Site_Health constructor.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param Experiments $experiments Experiments instance.
+	 *
+	 * @return void
+	 */
+	public function __construct( Experiments $experiments ) {
+		$this->experiments = $experiments;
+	}
+
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed() {
+		return is_admin() && ! wp_doing_ajax();
+	}
+
+	/**
+	 * Get the action to use for registering the service.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return string Registration action to use.
+	 */
+	public static function get_registration_action() {
+		return 'wp_loaded';
+	}
+
+	/**
+	 * Adds the filters.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @return void
+	 */
+	public function register() {
+		add_filter( 'debug_information', [ $this, 'add_debug_information' ] );
+		add_filter( 'site_status_test_php_modules', [ $this, 'add_extensions' ] );
+		add_filter( 'site_status_test_result', [ $this, 'modify_test_result' ] );
+	}
+
+	/**
+	 * Adds debug information for Web stories.
+	 *
+	 * @since 1.8.0
+	 *
+	 * @param array $debugging_information The debugging information from Core.
+	 *
+	 * @return array The debugging information, with added information for Web stories.
+	 */
+	public function add_debug_information( array $debugging_information ) {
+		$extra_data = [
+			'web_stories'             => [
+				'label'       => esc_html__( 'Web Stories Constants', 'web-stories' ),
+				'description' => esc_html__( 'Debugging information for the Web Stories for WordPress.', 'web-stories' ),
+				'fields'      => [
+					'web_stories_version'        => [
+						'label'   => 'WEBSTORIES_VERSION',
+						'value'   => WEBSTORIES_VERSION,
+						'private' => false,
+					],
+					'web_stories_db_version'     => [
+						'label'   => 'WEBSTORIES_DB_VERSION',
+						'value'   => WEBSTORIES_DB_VERSION,
+						'private' => false,
+					],
+					'web_stories_amp_version'    => [
+						'label'   => 'WEBSTORIES_AMP_VERSION',
+						'value'   => WEBSTORIES_AMP_VERSION,
+						'private' => false,
+					],
+					'web_stories_cdn_url'        => [
+						'label'   => 'WEBSTORIES_CDN_URL',
+						'value'   => WEBSTORIES_CDN_URL,
+						'private' => false,
+					],
+					'web_stories_dev_mode'       => [
+						'label'   => 'WEBSTORIES_DEV_MODE',
+						'private' => false,
+						'value'   => $this->get_formatted_output( WEBSTORIES_DEV_MODE ),
+						'debug'   => WEBSTORIES_DEV_MODE,
+					],
+					'web_stories_theme_support'  => [
+						'label'   => 'Theme supports',
+						'value'   => $this->get_formatted_output( current_theme_supports( 'web-stories' ) ),
+						'private' => false,
+					],
+					'web_stories_libxml_version' => [
+						'label'   => 'libxml Version',
+						'value'   => LIBXML_DOTTED_VERSION,
+						'private' => false,
+					],
+				],
+			],
+			'web_stories_experiments' => [
```

Comment:
Can we make this part of the above section instead?

I don't want to clutter Site Health with two large sections for one plugin. Combining them would keep things a bit tidier.

Then the field label would be "Enabled experiments" and the enabled experiments would be listed on the right in a `<ul>`.

Mock:

![Screenshot 2021-05-19 at 01 26 28](https://user-images.githubusercontent.com/841956/118735699-3ebeaa80-b841-11eb-99c8-18111b6dedda.png)